src/cmd/linuxkit/run_virtualizationframework.go

```diff
@@ -9,5 +9,5 @@ import (
 
 // Process the run arguments and execute run
 func runVirtualizationFramework(args []string) {
-	log.Fatal("virtualization framework is available only on macOS")
+	log.Fatal("virtualization framework is available only on macOS with CGO-enabled build of linuxkit")
```

Comment:
Here is the only change I would think you would need. Someone who didn't build linuxkit but runs it, will get confused. "It didn't work; what do I do now?"

It might be helpful if the message said something like, "'linuxkit run' on macOS requires virtualization framework. This build of linuxkit was compiled without virtualization framework capabilities. To perform 'linuxkit run' on macOS, please use a version of linuxkit compiled with virtualization framework."

Or something like that.

I don't think we need to say anything about cgo here, since the user might not know or care, and just will get confused. Instead, we can say, "sorry, you are running a version without vf; go get a version with it." Then they (or whoever they turn to to build it) will look at the docs you modified above to rebuild it, and see "oh, I need cgo!"

Reply:
Make sense, thank you. I put another file which will show an error you wrote here in case of `darwin && !cgo` build flags.